### ISSUE TITLE
Error on startup on one of my computers.

### ISSUE THREAD

**Issue by Distortionist** [WyvernTKC/cpuminer-gr-avx2#74]
Here is what it says, but continues to go.

`[2021-11-13 17:21:42] Software does NOT match CPU features!
[2021-11-13 17:21:42] Please check if proper binaries are being used.
`

Full output.

`Detected Intel CPU:  Intel(R) Core(TM) i5-3470S CPU @ 2.90GHz
Available CPU Instructions: AVX SSE42
Starting sse42 variant of the binaries.

         **********  cpuminer-opt-gr 1.2.3  ***********
     A CPU miner with multi algo support and optimized for CPUs
     with AVX512, SHA and VAES extensions by JayDDee.
     with Ghostrider Algo by Ausminer & Delgon.
     Jay D Dee's BTC donation address: 12tdvfF7KmAsihBXQXynT6E6th2c2pByTT

     RTM 1.50% Donation

Prepared for Linux
CPU:        Intel(R) Core(TM) i5-3470S CPU @ 2.90GHz
SW built on Oct 22 2021 with GCC 10.3.0
CPU features:  AVX
SW features:   SSE4.2
Algo features: AVX2   VAES

[2021-11-13 17:21:42] Software does NOT match CPU features!
[2021-11-13 17:21:42] Please check if proper binaries are being used.
[2021-11-13 17:21:42] Tune config 'tune_config' loaded succesfully
[2021-11-13 17:21:42] Huge Pages set up successfully.
`

**Comment by Distortionist** [comment]
Sorry, I did upgrade but I noticed it says 1.2.3, I may have started the wrong one, the 1.2.4.1 version has the same error though. I can post a new log if you want but its only the version that is different.

**Comment by michal-zurkowski** [comment]
Yes. We do not provide just AVX binaries as it is not any different than SSE42.
In theory, your CPU should support AES but it might be disabled in the bios or by the OEM.
In your case everything it ok.
